step09/README.md

```diff
@@ -1,12 +1,12 @@
-# Step 09: Buffer The String Concatenation Slayer
+# Step 09: Buffer "The String Concatenation Slayer"
 
 In this lesson you will learn how to:
 
 - Concatenate strings using a Buffer
 
 ## Overview
 
-In this lesson we will be adding support for multiple lives to the application. We will update the collision tracking code to decrement the number of lives instead of setting lives to 0 on a collision. We will also add a player position reset to restart play after collision with a ghost. Finally, we will add Player emojis to the game scoreboard to track the number of lives remaining instead of displaying lives as an integer value.
+In this lesson we will be adding support for multiple lives to the application. We will update the collision tracking code to decrement the number of lives instead of setting lives to 0 on a collision. We will also add a player position reset to restart player after collision with a ghost. Finally, we will add Player emojis to the game scoreboard to track the number of lives remaining instead of displaying lives as an integer value.
```

Comment:
Instead of `We will also add a player position reset to restart player after collision with a ghost`

`We will also keep track of the starting player position and respawn the player at it if they die.`